docs/UserGuide.md

```diff
@@ -18,50 +18,50 @@ If this is the first time you are using this user guide, we highly recommend you
-        4. [**Delete a contact `delete`**](#delete-a-contact-delete)
-        5. [**Add a label to a contact `tag`**](#add-a-label-to-a-contact-tag)
-        6. [**Remove a label from a contact `untag`**](#remove-a-label-from-a-contact-untag)
-    4. [**Advanced Commands**](#advanced-commands)
-        1. [**Search Commands**](#search-commands)
-            1. [**Filter contacts by keywords `find`**](#filter-contacts-by-keywords-find)
-            2. [**Arrange contacts based on criteria `sort`**](#arrange-contacts-based-on-criteria-sort)
-        2. [**Storage Commands**](#storage-commands)
-            1. [**Save a copy of EduMate `save`**](#save-a-copy-of-edumate-save)
-            2. [**Load a copy of EduMate `load`**](#load-a-copy-of-edumate-load)
-        3. [**Meet Commands**](#meet-commands)
-            1. [**Suggest places to meet with your contacts `meet`**](#suggest-places-to-meet-with-your-contacts-meet)
-    5. [**Other Commands**](#other-commands)
-        1. [**Exit the application**](#exit-the-application)
-8. [**FAQ**](#faq)
-9. [**Summary**](#summary)
-    1. [**Prefixes**](#prefixes)
-    2. [**Command Summary**](#command-summary)
-10. [**Troubleshooting**](#troubleshooting)
-     1. [**How to check your Java version**](#how-to-check-your-java-version)
+* [**Using this Guide**](#using-this-guide)
+* [**Table of Contents**](#table-of-contents)
+* [**Overview**](#overview)
+  * [**What is EduMate?**](#what-is-edumate)
+  * [**Understanding the Symbols and Colours**](#understanding-the-symbols-and-colours)
+  * [**Glossary**](#glossary)
+* [**Quick Start**](#quick-start)
+  * [**System Requirements**](#system-requirements)
+  * [**Installation Instructions**](#installation-instructions)
+  * [**Try it out! (Recommended)**](#try-it-out-recommended)
+  * [**Getting Help**](#getting-help)
+* [**User Interface**](#user-interface)
+  * [**Person List**](#person-list)
+  * [**Profile Panel**](#profile-panel)
+  * [**Command Box**](#command-box)
+  * [**Command Response**](#command-response)
+* [**Set Your Own Profile**](#set-your-own-profile)
+* [**Commands**](#commands)
+  * [**How to interpret the command format**](#how-to-interpret-the-command-format)
+  * [**Arguments**](#arguments)
+  * [**Basic Commands**](#basic-commands)
+    * [**Add a contact `add`**](#add-a-contact-add)
+    * [**View a contact's profile `view`**](#view-a-contacts-profile-view)
+    * [**Edit a contact's details `edit`**](#edit-a-contacts-details-edit)
+    * [**Delete a contact `delete`**](#delete-a-contact-delete)
+    * [**Add a label to a contact `tag`**](#add-a-label-to-a-contact-tag)
+    * [**Remove a label from a contact `untag`**](#remove-a-label-from-a-contact-untag)
+* [**Advanced Commands**](#advanced-commands)
+  * [**Search Commands**](#search-commands)
+    * [**Filter contacts by keywords `find`**](#filter-contacts-by-keywords-find)
+    * [**Arrange contacts based on criteria `sort`**](#arrange-contacts-based-on-criteria-sort)
```

Comment:
It makes sense to remove the numbering altogether. Is it alright for you to change the DG in the same way as well?

Reply:
the DG no numbering whut in the first place @mynameizzhafeez